src/basic/errno-util.h

```diff
@@ -212,6 +212,13 @@ static inline bool ERRNO_IS_NEG_DEVICE_ABSENT(intmax_t r) {
 }
 _DEFINE_ABS_WRAPPER(DEVICE_ABSENT);
 
+/* Device is absent or "empty". We get -ENOMEDIUM from CD/DVD devices, also in VMs. */
+static inline bool ERRNO_IS_NEG_DEVICE_ABSENT_OR_EMPTY(intmax_t r) {
+        return ERRNO_IS_NEG_DEVICE_ABSENT(r) ||
+                r == -ENOMEDIUM;
```

Comment:
Then, why not to add -ENOMEDIUM to `ERRNO_IS_NEG_DEVICE_ABSENT()`?

Reply:
Yeah, I didn't want to do this. "Device absent" has a specific meaning and I don't think it's appropriate to assume that it should be changed.